mailmerge/api.py

```diff
@@ -236,8 +243,13 @@ def main(sample=False,
                 print(">>> sent message {} DRY RUN".format(i))
             else:
                 # Send message
-                sendmail(text, sender, recipients, config_filename)
-                print(">>> sent message {}".format(i))
+                result = sendmail(text, sender, recipients, config_filename)
+                if (result == 0):
```

Comment:
Consider moving the `try` block here, instead of returning a code from `sendmail()`.

Reply:
Can do.  That's an easy change.